src/Modules/Pages/templates/Frontend/Navigation.html.twig

```diff
@@ -1,13 +1,13 @@
 {% if pages|count > 0 %}
-  <ul class="{{ isSubNavigation ? 'dropdown-menu' : 'navbar-nav me-auto mb-2 mb-lg-0'}}">
+  <ul class="{{ isSubNavigation ? 'dropdown-menu' : 'navbar-nav me-auto mb-2 mb-lg-0' }}">
```

Comment:
please change the php to is_sub_navigation assignment

Reply:
fixed e889bf94b984da871a882505e244a2b56f1a6274